app/frontend/styles/_tag.scss

```diff
@@ -0,0 +1,38 @@
+@mixin tag($color: "blue", $filled: true) {
+  white-space: nowrap;
+
+  @if $filled {
+    background-color: govuk-colour($color);
+    color: govuk-colour("white");
+  } @else {
+    background-color: govuk-colour("white");
+    border: 3px solid govuk-colour($color);
+    color: govuk-colour($color);
+    margin-top: -3px;
+    margin-bottom: -3px;
+  }
+}
```

Comment:
@fofr mentioned this may be pulled in from the design system; as an experiment, can you try to see if anything blows up if it's removed?

Reply:
It blows up when I remove it. Webpacker doesn't compile. :(

Reply:
No worries then!